travis/test_server.py

```diff
@@ -302,7 +298,13 @@ def main(argv=None):
             test_loghandler = 'openerp.tools.yaml_import:DEBUG'
     odoo_full = os.environ.get("ODOO_REPO", "odoo/odoo")
     server_path = get_server_path(odoo_full, odoo_version, travis_home)
-    script_name = get_server_script(odoo_version)
+    # Normalize on v10 odoo-bin command
+    try:
+        shutil.copy(
+            os.path.join(server_path, 'openerp-server'),
+            os.path.join(server_path, 'odoo-bin'))
```

Comment:
I have a concern about this copy operation: it is intrusive, in the sense that it makes modification on the code being used for the tests. This is will not only be unexpected, it will also be an issue when MQT will be used to run tests on our local dev machine. It changes a "stateless" operation into a "stateful" one, for no benefit at all.

Again, why is this change necessary?